runtime/bundles/org.eclipse.core.tools/src/org/eclipse/core/tools/search/FindUnusedMembers.java

```diff
@@ -202,7 +222,9 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 		try {
 			process(monitor);
 		} catch (OperationCanceledException e) {
-			throw new InterruptedException();
+			InterruptedException interruptedException = new InterruptedException();
```

Comment:
Is there a reason why the `InterruptedException` is not initialized with the message of the causing exception here, like it is done at the other places where this kind of exception wrapping is appleid?

Reply:
No reason, it's been added in c116573c4620558fc49a8b18999dfad4bc265c57.
Thank you for noticing!